crates/wal-protocol/src/lib.rs

```diff
@@ -246,6 +250,7 @@ pub enum Error {
 /// todo: This method should be removed in favor of using Appender/BackgroundAppender API in
 /// Bifrost. Additionally, the check for partition_table is probably unnecessary in the vast
 /// majority of call-sites.
+#[cfg(feature = "serde")]
 pub async fn append_envelope_to_bifrost(
```

Comment:
Can we move this method to the `bifrost` crate? Then we can get rid of the circular dependency and the cfg feature here.

Reply:
@tillrohrmann  fixed, ptal. (i didn't have permissions to add reviewer